frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx

```diff
@@ -203,13 +204,15 @@ export const ResourceInventoryItem: React.FC<ResourceInventoryItemProps> = ({
   ExpandedComponent,
   basePath,
 }) => {
-  const TitleComponent = React.useCallback(
+  let Title: React.ComponentType = React.useCallback(
```

Comment:
```suggestion
  let Title = React.useCallback(
```
NIT: type is not required since it's implicit from the initial value.